components.json

```diff
@@ -1078,6 +1078,11 @@
 			"owner": "mAAdhaTTah",
 			"require": "toolbar",
 			"noCSS": true
+		},
+		"diff-highlight": {
+			"title": "Diff Highlight",
+			"owner": "RunDevelopment",
+			"require": "diff"
```

Comment:
I'm surprised that this actually worked, well, at least it did on the download page.

As a more general question should a plugin be able to require a language like this?

Reply:
I don't see why not. It already works, it has a use case, why disallow it?

Reply:
Because you don't know whether it requires a plugin named `diff` or a language.

Reply:
Don't get me wrong. It's super useful that it works but I don't like that it's ambiguous.

Reply:
I think the likelihood that we have an plugin and language with the same name is basically nil. I don't think I mind that ambiguity. Maybe it should be documented somewhere?

Reply:
I'm planning a big dependency update for #1992. I'll document it then.